app/pings.yaml

```diff
@@ -43,3 +43,14 @@ topsites-impression:
     - https://github.com/mozilla-mobile/fenix/pull/23945
   notification_emails:
     - android-probes@mozilla.com
+
+spoc:
```

Comment:
I'd recommend adding "reasons" to this ping, and use them to distinguish between click and impression. See [this as an example](https://searchfox.org/mozilla-central/rev/968bd894205cf4f579d94ac4e175cc3187458605/browser/components/newtab/pings.yaml#13-20).